docs/semgrep-secrets/getting-started.md

```diff
@@ -0,0 +1,233 @@
+---
+slug: /semgrep-secrets/getting-started
+append_help_link: true
+title: Getting started with Semgrep Secrets
+hide_title: true
+description: Set up secrets scanning in Semgrep to find and rotate valid leaked secrets.
+tags:
+  - Semgrep Secrets
+toc_max_heading_level: 2
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Getting started with Semgrep Secrets
+
+Detect and triage leaked secrets and credentials by scanning with Semgrep Secrets.
+
+Save time by prioritizing which secrets to rotate based on whether they are valid (active and in-use) or not.
+
+![Semgrep Secrets page](/img/secrets-page.png)
+**_Figure._** Semgrep Secrets page. <br />
+
+This document guides you through the following:
+
+* Enabling Semgrep Secrets
+* Receiving findings in GitHub PR comments
+* Triaging findings
+
+To learn about how Semgrep Secrets detects secrets, see [<i class="fa-regular fa-file-lines"></i> Conceptual overview of Semgrep Secrets](/docs/semgrep-secrets/conceptual-overview).
```

Comment:
If you agree to the title change comment on the other file, this would also change.